File: week-02/day-01/28_draw_triangle.py
# Write a program that reads a number from the standard input, then draws a
# triangle like this:
#
# *
# **
# ***
# ****
#
# The triangle should have as many lines as the number was

num = input("Enter a number: ")
num = int(num)
str = "*"

for i in range(num):
    print(str)
    str += "*"

File: week-07/day-01/28_draw_triangle.js
'use strict';

let lineCount = 4;
let star = '*';

// Write a program that draws a
// triangle like this:
//
// *
// **
// ***
// ****
//
// The triangle should have as many lines as lineCount is

for (let i = 0; i < lineCount; i++) {
    console.log(star);
    star += '*';
}
